config/v1/console/types_console_server.go

```diff
@@ -0,0 +1,46 @@
+package console
+
```

Comment:
This seems to be the only resource that should exist under `/config/console`.  The rest should likely be nested under `/config/console/extensions`.